.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Shouldn't this be ignored?

Reply:
Not necessarily. This file is helpful for vscode users as it sets up the linters to run automatically.